include/randomizer_8_16_64.h

```diff
@@ -2,4 +2,6 @@
 #define RANDOMIZER_8_16_64_H
 
 int randomizer_8_16_64(int **input);
+void show_8_16_64(int **input);
+void free_8_16_64(int **input);
 #endif
```

Comment:
Da die funktionen show und free teil der Debugger implementierung wurden habe ich diese mit ins headerfile eingebaut.

Die neue Namensgebung kommt daher weil in Teil2 ähnlich input Array verwendet werden -> damit nicht zur verwechlung kommt habe ich sie daher spezifiziert.

Reply:
Gut gemacht.
Ich hab in meinem Commit jetzt in jedes File zu Beginn ein paar Kommentarzeilen für Doxygen eingebaut, viell. kannst du das in einer weiteren Version dann auch gleich reintun (vorerst mal fast copy und paste zu dem was ich zB ins File menu.c und menu.h gepostet hab) thx

Reply:
Hey Andi, ja kann ich gerne machen! Ich werd dann gleich einen neuen commit pushen :+1: